external-crates/move/move-compiler/tests/move_check/typing/constraints_not_satisfied_type_annotation.exp

```diff
@@ -21,3 +21,15 @@ error[E05001]: ability constraint not satisfied
   │                          │    The type '0x8675309::M::R' does not have the ability 'copy'
   │                          'copy' constraint not satisifed
 
+warning[W09004]: unnecessary trailing semicolon
+  ┌─ tests/move_check/typing/constraints_not_satisfied_type_annotation.move:7:35
+  │
+7 │         ignore((abort 0: CupC<R>));
+  │         --------------------------^
+  │         │                         │
+  │         │                         Invalid trailing ';'
+  │         │                         A trailing ';' in an expression block implicitly adds a '()' value after the semicolon. That '()' value will not be reachable
+  │         Any code after this expression will not be reached
```

Comment:
Depending on the type of `ignore`, this code may or maynot compile once you remove the `;`
We should only give this error in cases like 
`return 0` 

Really we have the dead code warning already `Invalid use of a divergent expression.` above